hack/terraform-quickstart/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "bootstrap_node_ip" {
+  value = "${aws_instance.bootstrap_node.public_ip}"
+}
+
+output "worker_ips" {
```

Comment:
I want this to be an array of IPs for the start-cluster.sh script to consume. So if I add the `count` field to the worker resource I think this will work out.